We do not have furo packaged yet.  It seems to be not needed to see tests pass.

--- greenlet-3.1.0/setup.py.orig
+++ greenlet-3.1.0/setup.py
@@ -248,7 +248,6 @@
     extras_require={
         'docs': [
             'Sphinx',
-            'furo',
         ],
         'test': [
             'objgraph',
